app-leechcraft/lc-poshuku/lc-poshuku-9999.ebuild

```diff
@@ -1,49 +1,52 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI="4"
+EAPI=6
 
-inherit confutils leechcraft
+inherit leechcraft
 
 DESCRIPTION="Poshuku, the full-featured web browser plugin for LeechCraft"
 
 SLOT="0"
 KEYWORDS=""
-IUSE="+autosearch debug +dcac +cleanweb +fatape +filescheme +fua +idn +keywords +onlinebookmarks
-		+pcre postgres qrd +sqlite wyfv"
+IUSE="+autosearch debug +dcac +cleanweb +fatape +filescheme +foc +fua +idn +keywords +onlinebookmarks
+	  postgres qrd +speeddial +sqlite webengineview +webkitview"
```

Comment:
Please rather use `webengine` and `webkit`, flags that are already in use by other packages in the tree.